src/main/java/step1/StringAddCalculator.java

```diff
@@ -0,0 +1,27 @@
+package step1;
+
+import java.util.List;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String given){
+        if(isNullinitValue(given)){
+            return 0;
+        }
+        Numbers numbers = new Numbers(given);
+        List<Integer> list = numbers.getNumbers();
+
+        int result = 0;
+        for (Integer integer : list) {
+            result += integer;
+        }
+        return result;
+    }
+
+    public static boolean isNullinitValue(String given){
+        if(given == null || given.isEmpty()){
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return given == null || given.isEmpty();
```
간결하게 표현할 수 있을지 살펴보면 좋겠습니다. 😀

Reply:
네 앞으로도 boolean 값을 반환하는 경우엔 인라인으로 더 간단하게 해보겠습니다!